doc/source/regression/modes.md

```diff
@@ -13,32 +13,12 @@ kernelspec:
 
 +++ {"tags": ["jupyterlite_sphinx_strip"]}
 
-```{eval-rst}
-.. currentmodule:: pywt
-
-.. dropdown:: 🧑‍🔬 This notebook can be executed online. Click this section to try it out! ✨
-    :color: success
-
-    .. notebooklite:: modes.ipynb
-      :width: 100%
-      :height: 600px
-      :prompt: Open notebook
-
-.. dropdown:: Download this notebook
-    :color: info
-    :open:
-
-    Please use the following links to download this notebook in various formats:
-
-    1. :download:`Download IPyNB (IPython Notebook) <modes.ipynb>`
-    2. :download:`Download Markdown Notebook (Jupytext) <modes.md>`
+```{include} header.md
 ```
 
-+++
-
 # Signal Extension Modes
 
-Let's import `pywt`, first:
+Import `pywt` first:
```

Comment:
I remember coming across a style guide, maybe internal to Quansight or one that folks were citing, that said to just use imperative sentences -- "do x" rather than "let's do x"

Reply:
That is interesting. I would love to see it sometime if you can find it!

Reply:
I do not know how I would find it, but for example, [Google's guidelines say to use the second person](https://developers.google.com/style/person)